ownCloud File Provider/FileProviderExtension.m

```diff
@@ -45,20 +45,57 @@ - (instancetype)init
 	OCAppIdentity.sharedAppIdentity.appGroupIdentifier = bundleInfoDict[@"OCAppGroupIdentifier"];
 
 	if (self = [super init]) {
-		_fileManager = [[NSFileManager alloc] init];
+		_fileManager = [NSFileManager new];
```

Comment:
@felix-schwarz I find alloc / init more explicit but it's a matter of personal preference I guess

Reply:
Until recently, I used to be in the `alloc` / `init` camp, too. But since I started writing more and more Swift code, `alloc` / `init` reminded me of `allocWithZone:` and felt outdated. FWIW `new` has also been around since macOS 10.0. It really boils down to personal preference as you say. :)